Fix MSVC build: VASP Format

During compilation:
<pre>
C:\openbabel\src\formats\vaspformat.cpp(488): error C2039: 'bind2nd' : is not a member of 'std' [C:\openbabel\build\src\formats\formats_compchem.vcxproj]
C:\openbabel\src\formats\vaspformat.cpp(488): error C2039: 'not_equal_to' : is not a member of 'std' [C:\openbabel\build\src\formats\formats_compchem.vcxproj]
C:\openbabel\src\formats\vaspformat.cpp(488): error C2065: 'not_equal_to' : undeclared identifier [C:\openbabel\build\src\formats\formats_compchem.vcxproj]
C:\openbabel\src\formats\vaspformat.cpp(488): error C2062: type 'double' unexpected [C:\openbabel\build\src\formats\formats_compchem.vcxproj]
C:\openbabel\src\formats\vaspformat.cpp(488): error C3861: 'bind2nd': identifier not found [C:\openbabel\build\src\formats\formats_compchem.vcxproj]
C:\openbabel\src\formats\vaspformat.cpp(542): error C2039: 'bind2nd' : is not a member of 'std' [C:\openbabel\build\src\formats\formats_compchem.vcxproj]
C:\openbabel\src\formats\vaspformat.cpp(542): error C2039: 'divides' : is not a member of 'std' [C:\openbabel\build\src\formats\formats_compchem.vcxproj]
C:\openbabel\src\formats\vaspformat.cpp(542): error C2065: 'divides' : undeclared identifier [C:\openbabel\build\src\formats\formats_compchem.vcxproj]
C:\openbabel\src\formats\vaspformat.cpp(542): error C2062: type 'double' unexpected [C:\openbabel\build\src\formats\formats_compchem.vcxproj]
C:\openbabel\src\formats\vaspformat.cpp(542): error C3861: 'bind2nd': identifier not found [C:\openbabel\build\src\formats\formats_compchem.vcxproj
</pre>

--- a/src/formats/vaspformat.cpp
+++ b/src/formats/vaspformat.cpp
@@ -17,6 +17,7 @@ GNU General Public License for more details.
 
 #include <limits.h>
 #include <locale> // For isalpha(int)
+#include <functional>
 
 #define EV_TO_KCAL_PER_MOL 23.060538
 

